tests/suites/test_suite_psa_crypto.function

```diff
@@ -3152,6 +3152,862 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void aead_multipart_encrypt( int key_type_arg, data_t *key_data,
+                             int alg_arg,
+                             data_t *nonce,
+                             data_t *additional_data,
+                             int ad_part_len,
+                             data_t *input_data,
+                             int data_part_len,
+                             data_t *expected_result )
```

Comment:
`expected_result` is not precise enough: the multipart API outputs the tag separately, but the test function doesn't have enough information to detect if the implementation outputs the part of the tag with the ciphertext/plaintext output or vice versa.

Reply:
I am not sure what you are asking for here - do you want me to pass in the expected tag seperately and thus check it seperately?